SPECS/dracut/dracut.spec

```diff
@@ -96,96 +109,90 @@ cp %{SOURCE1} .
 %make_build
 
 %install
-%make_install \
-     DESTDIR=%{buildroot} \
-     libdir=%{_libdir}
+%make_install %{?_smp_mflags} libdir=%{_libdir}
 
-echo "DRACUT_VERSION=%{version}-%{release}" > %{buildroot}/%{dracutlibdir}/dracut-version.sh
+echo "DRACUT_VERSION=%{version}-%{release}" > %{buildroot}%{dracutlibdir}/%{name}-version.sh
 
-rm -fr -- %{buildroot}/%{dracutlibdir}/modules.d/00bootchart
 
 # we do not support dash in the initramfs
-rm -fr -- %{buildroot}/%{dracutlibdir}/modules.d/00dash
+rm -fr -- %{buildroot}%{dracutlibdir}/modules.d/00dash
 
 # remove gentoo specific modules
-rm -fr -- %{buildroot}/%{dracutlibdir}/modules.d/50gensplash
+rm -fr -- %{buildroot}%{dracutlibdir}/modules.d/96securityfs \
+          %{buildroot}%{dracutlibdir}/modules.d/97masterkey \
+          %{buildroot}%{dracutlibdir}/modules.d/98integrity
 
-rm -fr -- %{buildroot}/%{dracutlibdir}/modules.d/96securityfs
-rm -fr -- %{buildroot}/%{dracutlibdir}/modules.d/97masterkey
-rm -fr -- %{buildroot}/%{dracutlibdir}/modules.d/98integrity
-
-mkdir -p %{buildroot}/boot/dracut
-mkdir -p %{buildroot}%{_sharedstatedir}/dracut/overlay
-mkdir -p %{buildroot}%{_localstatedir}/log
-touch %{buildroot}%{_localstatedir}/log/dracut.log
-mkdir -p %{buildroot}%{_sharedstatedir}/initramfs
-
-rm -f %{buildroot}%{_mandir}/man?/*suse*
+mkdir -p %{buildroot}/boot/%{name} \
+         %{buildroot}%{_sharedstatedir}/%{name}/overlay \
+         %{buildroot}%{_var}/log \
+         %{buildroot}%{_var}/opt/%{name}/log \
+         %{buildroot}%{_sharedstatedir}/initramfs \
+         %{buildroot}%{_sbindir}
 
 install -m 0644 dracut.conf.d/fips.conf.example %{buildroot}%{_sysconfdir}/dracut.conf.d/40-fips.conf
 > %{buildroot}%{_sysconfdir}/system-fips
 
-install -m 0755 %{SOURCE2} %{buildroot}%{_bindir}/mkinitrd
+install -m 0755 %{SOURCE1} %{buildroot}%{_bindir}/mkinitrd
 
-install -m 0644 %{SOURCE3} %{buildroot}%{_sysconfdir}/dracut.conf.d/50-megaraid.conf
+install -m 0644 %{SOURCE2} %{buildroot}%{_sysconfdir}/dracut.conf.d/50-megaraid.conf
 
 mkdir -p %{buildroot}%{_libdir}/dracut/modules.d/20overlayfs/
+install -p -m 0755 %{SOURCE3} %{buildroot}%{_libdir}/dracut/modules.d/20overlayfs/
 install -p -m 0755 %{SOURCE4} %{buildroot}%{_libdir}/dracut/modules.d/20overlayfs/
-install -p -m 0755 %{SOURCE5} %{buildroot}%{_libdir}/dracut/modules.d/20overlayfs/
 
-# create compat symlink
-mkdir -p %{buildroot}%{_sbindir}
-ln -sr %{buildroot}%{_bindir}/dracut %{buildroot}%{_sbindir}/dracut
+touch %{buildroot}%{_var}/opt/%{name}/log/%{name}.log
+ln -srv %{buildroot}%{_var}/opt/%{name}/log/%{name}.log %{buildroot}%{_var}/log/
 
-%check
-%make_build -k clean check
+# create compat symlink
+ln -srv %{buildroot}%{_bindir}/%{name} %{buildroot}%{_sbindir}/%{name}
 
 %files
```

Comment:
Seems we have lost the `%license` entry here.

Reply:
this was removed from the photon version (and i don't see why we need to install a license file)

Reply:
i still dont see why we need this but i added it back